NGS2_WITNESS_Cycle1_MASTER.R

```diff
@@ -0,0 +1,59 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. on August 2017
+
+### System information
+
+a=Sys.info()
+print(a)
+b=sessionInfo(package = NULL)
+print(a)
+
+### Set base directory
+
+setwd(X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1)

```

Comment:
Pablo, the issue is for Mac users, there is no X drive. So, the equivalent path for Anu and I, for example, is `/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1`. So, to Anu's point, the following should be good:

```
a = Sys.info()[1]
if( a == "Windows") { setwd("X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
if( a != "Windows") { setwd("/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
```
That way, if Anu or I were to try and execute anything we wouldn't get an error.